src/cli/interaction.rs

```diff
@@ -19,8 +19,8 @@ pub(crate) async fn confirm(question: impl AsRef<str>, default: bool) -> eyre::R
     ",
         question = question.as_ref(),
         are_you_sure = "Proceed?".bright_white().bold(),
-        no = "N".red().bold(),
-        yes = "y".green(),
+        no = if default {"n"} else {"N"}.red(),
```

Comment:
I removed bolding here, because I assume it's intended to emphasise the default and the type checker doesn't like `if default {"n"} else {"N".bold()}`. Any elegant solutions for this?